app/models/resource.rb

```diff
@@ -1,10 +1,12 @@
 class Resource < ActiveRecord::Base
   belongs_to :conference
-  validate :used_less_than_quantity
+  validates :name, presence: true
+  validates :used, :quantity, presence: true, numericality: { greater_than_or_equal_to: 0, only_integer: true }
```

Comment:
I'd rather you add all the attributes that need to be present in one validation, meaning move the presence validation in the above line, instead of validating for presence in more than one place.
That way we can easily see in 1 validation which are the attributes that need to be present.